### ISSUE TITLE
Help for extremely unsavvy user

### ISSUE THREAD

**Issue by virtu0s0** [ethereum/mist#2266]
``````
Version: `0.5.1`
OS & Version: windows 7 64bit
Node type: `eth/geth(default)`  ----not sure?
``````

Guys, I need your help big time. I am not a computer savvy person at all. I started working for a client that paid me in bitcoin, and last year after reading about ethereum, I decided to buy some. I downloaded mist 0.5.1 in march of last year and bought 175 eth before the fork. I have not touched them since.

My computer is very old and I would like to upload them to an exchange.  My blockchain was about 2 million blocks behind and updating extremely slowly. I decided to delete my chaindata and try to redownload it the fast way. I have only gotten up to block 2.2million, with about 1.3 million blocks to go, but the speed has dropped significantly. 

And now, when I do the skip peer search and look in my mist, it displays my wallet addresses, but where it used to show the eth that I have, it just shows 0.00. I used etherscan.io to make sure the money is still on those addresses, and there haven't been any outgoing transactions.

Basically, I don't know what I don't even know. Do I need to download a newer version of mist? I am very nervous about doing something wrong because I don't want to lose my wallet data. I have my wallet and password backed up. However, I don't know what the 'master password' is to change node from geth to eth. I don't even know what that does. 

If someone is willing to help me, I am willing to give you a fee, either in eth or etc for helping me recover and securely move my eth. We can go step by step and I can upload screenshots, etc. 

As I said, I am very confused and now very worried that my balance displays as 0.00

Thank you
J

**Comment by garretlaxton** [comment]
Your balance shows 0 because you deleted the chain data folder which has all the blocks, you will need to let Mist sync for your funds to reflect. If you wanna access to your ETH instantly you can import your backup here myetherwallet.com and you can view your balance and send if you'd like.
